browser/src/layer/marker/Cursor.ts

```diff
@@ -225,19 +225,26 @@ class Cursor {
 	}
 
 	private updatePositionAndSize() {
-		let diffX = -app.activeDocument.activeView.viewedRectangle.pX1;
-		let diffY = -app.activeDocument.activeView.viewedRectangle.pY1;
+		let x;
+		let y;
 
 		if (app.map.getDocType() === 'spreadsheet') {
+			let diffX = -app.activeDocument.activeView.viewedRectangle.pX1;
+			let diffY = -app.activeDocument.activeView.viewedRectangle.pY1;
+
 			if (app.isXOrdinateInFrozenPane(this.rectangle.pX1))
 				diffX = 0;
 
 			if (app.isYOrdinateInFrozenPane(this.rectangle.pY1))
 				diffY = 0;
-		}
 
-		let x = Math.round((this.rectangle.pX1 + diffX + app.sectionContainer.getDocumentAnchor()[0]) / app.dpiScale);
-		let y = Math.round((this.rectangle.pY1 + diffY + app.sectionContainer.getDocumentAnchor()[1]) / app.dpiScale);
+			x = Math.round((this.rectangle.pX1 + diffX + app.sectionContainer.getDocumentAnchor()[0]) / app.dpiScale);
+			y = Math.round((this.rectangle.pY1 + diffY + app.sectionContainer.getDocumentAnchor()[1]) / app.dpiScale);
+		}
+		else {
+			x = Math.round(this.rectangle.v1X / app.dpiScale);
```

Comment:
Nit: it would be nice to document what v1X/Y (i.e. the "v" variant, over pX and cX) is at the top of geometry.ts. Is "v" also CSS pixels, i.e. "v" and "c" the same?

See https://github.com/CollaboraOnline/online/blob/8a02914c9b4062e60d96639324e49a7449ca6fdb/browser/src/core/geometry.ts#L42-L43 where "p" and "c" is documented.

Reply:
That's right, here we have the explanations, in the last commit:
* https://github.com/CollaboraOnline/online/pull/13689